### PR TITLE
React: Different texts in top up with fiat

### PR DESCRIPTION
**Pull request by cehali**
# Summary
Show different text when topping up with fiat, depending on the provider.

Fixes: 0

## Checklist
- [x] Change is small and easy to review (please split big changes into multiple PRs)
- [x] Change is consistent with architecture
- [x] Change is consistent with test architecture
- [x] Change is consistent with naming conventions
- [x] New code is covered with tests
- [x] Tests related to old code are updated
- [x] Documentation is up to date with changes

### CODE REVIEW

File: universal-login-react/src/ui/TopUp/Fiat/TopUpWithFiat.tsx [hunk 1]
```diff
@@ -57,7 +57,8 @@ export const TopUpWithFiat = ({sdk, onPayClick, logoColor}: TopUpWithFiatProps)
   useAsyncEffect(recognizeUserCountry, []);
 
   const isPayButtonDisabled = !paymentMethod ||
-    (topUpProviderSupportService.isInputAmountUsed(paymentMethod) && !amount);
+    (topUpProviderSupportService.isInputAmountUsed(paymentMethod) && !amount) ||
+    paymentMethod === TopUpProvider.WYRE;
```
Comment (rzadp) [inline universal-login-react/src/ui/TopUp/Fiat/TopUpWithFiat.tsx]: Seems like it should be in a separate PR?

Comment (cehali) [reply universal-login-react/src/ui/TopUp/Fiat/TopUpWithFiat.tsx]: Done

**Comment by cehali** [comment]
image

**Comment by cehali** [comment]
image

**Comment by cehali** [comment]
image

**Comment by cehali** [comment]
image

**Comment by cehali** [comment]
image

**Comment by cehali** [comment]
image

File: universal-login-react/src/ui/TopUp/Fiat/FiatPaymentMethods.tsx [hunk 2]
```diff
@@ -12,45 +12,60 @@ interface FiatPaymentMethodsProps {
...
+  <TopUpRadio
+    checked={paymentMethod === TopUpProvider.WYRE}
+    onClick={() => setPaymentMethod(TopUpProvider.WYRE)}
+    name="payment-method"
+    className="fiat-payment-method"
+  >
+    <img src={providerLogo} srcSet={providerLogo} alt="wyre" className="wyre-logo"/>
+  </TopUpRadio>
+);
+
```
Comment (xitronix) [inline universal-login-react/src/ui/TopUp/Fiat/FiatPaymentMethods.tsx]: What do you think about: 
```ts
const ProviderTopUp = ({providerLogo, setPay, currentPaymentMethod, topUpProvider}: SomeProps) => (
  <TopUpRadio
    checked={currentPaymentMethod === topUpProvider}
    onClick={() => setPaymentMethod(topUpProvider)}
    name="payment-method"
    className="fiat-payment-method"
  >
    <img src={providerLogo} srcSet={providerLogo} alt="topUpProvider.toLowerCase()" className=`${topUpProvider.toLowerCase()}-logo`/>
  </TopUpRadio>
)
```

Comment (cehali) [reply universal-login-react/src/ui/TopUp/Fiat/FiatPaymentMethods.tsx]: 👍

Comment (cehali) [reply universal-login-react/src/ui/TopUp/Fiat/FiatPaymentMethods.tsx]: Done
